docs/ConfigureScalarDL.md

```diff
@@ -2,156 +2,84 @@
 
 This guide explains how to configure Scalar DL custom values in helm charts for Scalar DL deployment.
 
-## Create Kubernetes Secrets
+## Update Configuration Files
```

Comment:
Is it correct?
I think the structure is not correct.

Reply:
Suggestion (Don't follow blindly)

# Configure Scalar DL
## Reference values
## Create kubernetes secrets
## Update configuration files
### Configure schema loading custom value file
### Configure Scalar DL Ledger custom value file
### Configure Scalar DL Auditor custom value file
### Enable Auditor

Reply:
Thank you for the suggestion.
I will revise it to be more clear.